ydb/docs/en/core/reference/ydb-sdk/health-check-api.md

```diff
@@ -38,37 +49,98 @@ message IssueLog {
+| Parameter | Description |
+|:----|:----|
+| `ReturnVerboseStatus` | as was said earlier this parameter affects the filling of `database_status` field. |
+| `MinimumStatus` | the minimum status that will be given in the response. Issues with a better status will be discarded. |
+| `MaximumLevel` | maximum depth of issues in response. Deeper levels will be discarded |
 
 ## Possible problems {#problems}
 
-* `Pool usage over 90/95/99%`: One of the pools' CPUs is overloaded.
-* `System tablet is unresponsive / response time over 1000ms/5000ms`: The system tablet is not responding or it takes too long to respond.
-* `Tablets are restarting too often`: Tablets are restarting too often.
-* `Tablets are dead`: Tablets are not started (or cannot be started).
-* `LoadAverage above 100%`: A physical host is overloaded.
-* `There are no compute nodes`: The database has no nodes to start the tablets.
-* `PDisk state is ...`: Indicates problems with a physical disk.
-* `PDisk is not available`: A physical disk is not available.
-* `Available size is less than 12%/9%/6%`: Free space on the physical disk is running out.
-* `VDisk is not available`: A virtual disk is not available.
-* `VDisk state is ...`: Indicates problems with a virtual disk.
-* `DiskSpace is ...`: Indicates problems with virtual disk space.
-* `Storage node is not available`: A node with disks is not available.
-* `Replication in progress`: Disk replication is in progress.
-* `Group has no redundancy`: A storage group lost its redundancy.
-* `Group failed`: A storage group lost its integrity.
-* `Group degraded`: The number of disks allowed in the group is not available.
+| Message | Description |
+|:----|:----|
+| **DATABASE** ||
+| `Database has multiple issues`</br>`Database has compute issues`</br>`Database has storage issues` | These issues depend solely on the underlying `COMPUTE` and `STORAGE` layers. This is the most general status of the database. |
+| **STORAGE** ||
+| `There are no storage pools` | Unable to determine `STORAGE_POOLS` issues below. |
+| `Storage degraded`</br>`Storage has no redundancy`</br>`Storage failed` | These issues depend solely on the underlying `STORAGE_POOLS` layer. |
+| `System tablet BSC didn't provide information` | Storage diagnostics will be generated with alternative way. |
+| `Storage usage over 75%/85%/90%` | Need to increase disk space. |
+| **STORAGE_POOL** ||
+| `Pool degraded/has no redundancy/failed` | These issues depend solely on the underlying `STORAGE_GROUP` layer. |
+| **STORAGE_GROUP** ||
+| `Group has no vslots` ||
+| `Group degraded` | The number of disks allowed in the group is not available. |
+| `Group has no redundancy` | A storage group lost its redundancy. |
+| `Group failed` | A storage group lost its integrity. |
+||`HealthCheck` checks various parameters (fault tolerance mode, number of failed disks, disk status, etc.) and, depending on this, sets the appropriate status and displays a message. |
```

Comment:
This row seems out of place as the first column is empty; what it is supposed to describe?

Reply:
It describes `Group degraded`, `Group has no redundancy`, `Group failed` in common.